libs/test-helpers.js

```diff
@@ -1,28 +1,12 @@
 ( function( angular ) {
   var sendKeys = function( value ) {
     this.val( value );
-
-    var e = $.Event( 'input' );
-    var e2 = $.Event( 'keydown' );
-    for ( var i = 0; i < value.length; i++ ) {
-      e.which = e2.which = value.charCodeAt( i );
-      this.trigger( e );
-      this.trigger( e2 );
-    }
-
-    // if value is blank send a backspace
-    if ( !value.length ) {
-      e = $.Event( 'input' );
-      e.which = 8;
-      this.trigger( e );
-    }
+    this.trigger( $.Event('change') );
```

Comment:
Whilst the existing code was nicer, in that it represented more accurately what was happening, it doesn't work in IE and I needed to run the test suite against IE11.